concepts/Common/Integration Guides/50 Use MUI components in your DevExtreme application/50 Use MUI components in your DevExtreme application.md

```diff
@@ -0,0 +1,191 @@
+# Use MUI components in DevExtreme apps
+
+MUI is a popular line of React components and design tools. The design of MUI components follows Google's Material Design guidelines. Most MUI products are open-source, and do not require a commercial license.
+
+You don't have to use MUI to add a Material Design look and feel to your application. The DevExtreme library includes [a range of  Material Design themes](https://devexpress.github.io/ThemeBuilder/master/material/blue-light). However, if you want to use both MUI components and DevExtreme components, you can.
+
+You don't need extra configuration to run MUI components side by side with DevExtreme components. But if you nest components from different frameworks, you have to understand how they interact with one another.
+
+- [Run the sample application](#run-the-sample-application)
```

Comment:
You have broken links here. If you want to create TOC, do it as in the following PR: [!5861](https://github.com/DevExpress/devextreme-documentation/pull/5861/files)